### DISCUSSION TITLE
Go Support

### DISCUSSION THREAD

**Discussion by tmc** [langchain-ai/langsmith-sdk#426] [Ideas]
It'd be great to see an implementation allowing [langchaingo](https://github.com/tmc/langchaingo)-based programs to integrate with langsmith.

**Comment by hinthornw** [comment]
Hi @tmc thank you for the request! I think it would be straightforward to create, though I'm not sure I would be able to actively maintain it. Is this something you'd be interested in collaborating on?

**Comment by pvora27** [comment]
I'm also interested in seeing langchaingo support!

**Comment by hinthornw** [comment]
If we get enough real interest we could add. Currently don't have capacity to maintain a crisp package for that though

**Comment by tmc** [comment]
I'd be open to maintaining this as part of langchaingo if developed.

**Comment by tmc** [comment]
I have a start of a Go langsmith SDK pushed at https://github.com/tmc/langsmith-sdk/tree/go-support

With a very basic/minimal use example in https://github.com/tmc/langsmith-sdk/blob/go-support/go/langsmith/langsmith_examples_test.go
